jans-cedarling/bindings/cedarling_python/src/authorize/request.rs

```diff
@@ -81,9 +81,9 @@ impl Request {
         })?;
 
         Ok(cedarling::Request {
-            access_token: self.access_token.clone(),
-            id_token: self.id_token.clone(),
-            userinfo_token: self.userinfo_token.clone(),
+            access_token: Some(self.access_token.clone()),
```

Comment:
Do you assume that using python binding we always have all tokens?

Reply:
fixed in [33bdfbf](https://github.com/JanssenProject/jans/pull/10436/commits/33bdfbf9c3d2e730efe797ba5ec88f81815e20aa)